content/careers/support-engineer/index.md

```diff
@@ -14,46 +14,69 @@ department: "Tech"
 
 ### About LocalStack
 
-{{< jobs/about-localstack >}}
+We are a fast-growing startup headquartered in Zurich, Switzerland. With a development office in Vienna, Austria, and team members, from around the world.
+
+The LocalStack team is creating world-class tooling to support highly efficient development and testing for cloud application developers. Our core product is the LocalStack cloud emulator, a lightweight cloud sandbox running in Docker that allows our users to develop cloud applications entirely on their local machines.
+
+LocalStack has a large and active open-source community (45k+ stars on GitHub) with several hundred thousand active users worldwide. We are on an exciting growth journey to become a global leader because of our growing international customer base and our growing, globally distributed team of top-notch engineers.
```

Comment:
why remove the re-usable include snippet instead of updating it?

Reply:
I was not sure if I can do it. I went the safe route and updated the text directly. 

If you are ok with updating the snippet, I will do it.

Reply:
it's OK, let's consolidate in a follow-up PR